driver/Commandline.ml

```diff
@@ -99,4 +100,9 @@ let parse_array spec argv first last =
   in parse first
 
 let parse_cmdline spec =
-  parse_array spec Sys.argv 1 (Array.length Sys.argv - 1)
+  try
+    let argv = expandargv Sys.argv in
+    parse_array spec argv 1 (Array.length argv - 1)
+  with Arg.Bad s ->
```

Comment:
Can the `Arg.Bad` exception be raised here?  I don't think so.  Perhaps you meant `Responsefile.Error`.